app/views/layouts/application.html.slim

```diff
@@ -54,7 +54,7 @@ html
 
 
     #main
-      - if !request.original_url.include? "/partners/"
+      - if request.path.starts_with?("/publisher")
```

Comment:
this would match something like `/publisher_foobar`, which i assume you don't want.

Reply:
This is just UI that we only want to show for publishers exclusively. I can't foresee any risk.